EnglishFilter/sections/general_extensions.txt

```diff
@@ -27,7 +27,7 @@ shameless.com#%#//scriptlet("abort-current-inline-script", "$LAB", "/getexoloade
 !
 ! 'PopJSLock'
 ! #%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "constructor")
-freepornxxxhd.com,givemenflstreams.com,givemenbastreams.com,givememmastreams.com,givemeredditstreams.com,shinden.pl,ozporno.com,area51deportiva.com,xxxfree.watch,linksly.co,animesanka.com,mhometheater.com#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "constructor")
+tmnlsmvgrkoj.com,xokikdassjjhrn.com,freepornxxxhd.com,givemenflstreams.com,givemenbastreams.com,givememmastreams.com,givemeredditstreams.com,shinden.pl,ozporno.com,area51deportiva.com,xxxfree.watch,linksly.co,animesanka.com,mhometheater.com#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "constructor")
```

Comment:
Why did you add those domains here?

Reply:
Check yourself why. Too much too explain.

Reply:
> Check yourself why. Too much too explain.

Be prepared for the fact that our team can ask you questions. This is not proof. I can't approve it without an explanation.

Reply:
Ok wait.. Let me send a screenshot.

Reply:
This scriptlet is for original domains, not the adservers.

Reply:
![image](https://user-images.githubusercontent.com/76908410/108188041-12b06200-714a-11eb-8010-abcd7f5ffc01.png)
@zloyden

Reply:
Ah yes, those adservers does the job. Quite forgot that.